flow2ml/Tf_Results.py

```diff
@@ -0,0 +1,129 @@
+from sklearn.metrics import roc_auc_score, roc_curve
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from sklearn.metrics import plot_confusion_matrix, roc_curve,auc
+from sklearn.metrics import plot_precision_recall_curve, precision_recall_curve
+from docx import Document
+from docx.shared import Inches
+import os
+
+import tensorflow as tf
+import seaborn as sn
+
+class Tf_Results:
+    def __init__(self,model,validtion_generator):
+        '''
+        Initialises the class variables.
+        '''
+        self.model = model          #stores the model
+        self.validation_generator = validtion_generator    
```

Comment:
What is validation_generator here. Please describe the same in params section in documentation part

Reply:
Validation generator is the validation dataset object generated by tensorflow Image generator. This something that solely belongs to the domain of knowledge of tensorflow users. I don't think we need to document this as it's available in tensorflow.

Reply:
Yes, thats understood
But we can’t assume things just like that.
As a part of documentation for functions, you should mention clearly what each argument meant

Reply:
Alright made it.